cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -84,6 +84,8 @@ spec:
             - --container=heapster
             - --poll-period=300000
             - --estimator=exponential
+          securityContext:
+            runAsNonRoot: false
```

Comment:
> They run as root, but they shouldn't

I guess what I'm getting at is was this change necessary to get these pod specs to pass PSP, or is this unrelated proactive tightening of their permissions?